README.md

```diff
@@ -22,20 +22,20 @@ Ensure the following is set up in your repository settings before enabling this
 
 - **“Allow auto-merge”** is enabled.
 
-
 ## Inputs
 
-| Name | Required  | Description |
-|------|-----------|-------------|
-| `token` | yes | A GitHub Token other than the default `GITHUB_TOKEN` needs to be specified in order to be able to enable auto-merge. |
-| `merge-method` | no | Specify which merge method to use. By default, will select the first one available in this order: `merge`, `squash`, `rebase` |
-| `squash-title` | no | Use the pull request title as the commit message when squashing. |
-| `do-not-merge-labels` | no | When any of the labels in this comma-separated list is applied to a pull request, it will not be merged automatically. |
-| `required-labels` | no | Comma-separated list of labels that are required to be applied to a pull request for it to be merged automatically. |
-| `pull-request` | no | Try merging the specified pull request automatically. For example, you can pass an input from a `workflow_dispatch` event. |
-| `pull-request-author-associations` | no | Comma-separated list of required author associations for the pull request author. (By default, pull requests by any author are allowed.)|
-| `dry-run` | no | If set to `true`, will not actually merge pull requests but still perform all other checks. |
-
+| Name                               | Required | Description                                                                                                                                                                                                       |
```

Comment:
sorry about the reformat here, I can make the change more concise but VSCode ended up reformatting the entire table by default and I figured not too much harm.